dcr/scenarios/agent-bvt/check_firewall.py

```diff
@@ -24,8 +24,8 @@ def __is_firewall_enabled():
                 if update_match:
                     return update_match.groups()[0].lower() == 'y'
 
-    # The firewall is enabled by default.
-    return True
+    # The firewall is disabled by default.
+    return False
```

Comment:
As seen with mariner 1.0, when there is no entry in the config file, the agent disables the firewall.